daemon/container.go

```diff
@@ -649,12 +637,11 @@ func (container *Container) ReleaseNetwork() {
 	if container.Config.NetworkDisabled || !container.hostConfig.NetworkMode.IsPrivate() {
 		return
 	}
-	eng := container.daemon.eng
 
-	job := eng.Job("release_interface", container.ID)
-	job.SetenvBool("overrideShutdown", true)
-	job.Run()
-	container.NetworkSettings = &NetworkSettings{}
+	_ = bridge.Release(container.ID)
+	//job.SetenvBool("overrideShutdown", true)
```

Comment:
@crosbymichael another comment that needs your attention :S It was there as a hack to make sure the job was run during engine shutdown. Not sure if we need something special.